### PR TITLE
Fixes bug 1269894 - Add an annotation to AMD CPU bug related crashes.

### PR DESCRIPTION
**Pull request by adngdb**
Here's what this looks like:

### CODE REVIEW

File: webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html [hunk 1]
```diff
@@ -181,7 +180,15 @@
                             </tr>
                             <tr>
                                 <th scope="row">Build Architecture Info</th>
-                                <td>{{ report.cpu_info }}</td>
+                                <td>
+                                    {% if report.cpu_info %}
+                                        {% if report.cpu_info.startswith('AuthenticAMD family 20 model 1') or
+                                              report.cpu_info.startswith('AuthenticAMD family 20 model 2') %}
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html]: what if there's a `AuthenticAMD family 20 model 3` in a couple of months time?

File: webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html [hunk 2]
```diff
@@ -181,7 +180,15 @@
...
+                                            <img src="{{ static('img/3rdparty/silk/exclamation.png') }}" alt="warning" title="Possible AMD CPU bug related crash report" />
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html]: There needs to be a comment that explains why we do this.

File: webapp-django/crashstats/signature/jinja2/signature/signature_reports.html [hunk 3]
```diff
@@ -17,9 +17,13 @@
             {% for crash in query.hits %}
             <tr>
                 <td>
-                    <a href="{{ url('crashstats:report_index', crash.uuid) }}" class="external-link crash-id">
-                        {{ crash.uuid }}
-                    </a>
+                    <a href="{{ url('crashstats:report_index', crash.uuid) }}" class="external-link crash-id">{{ crash.uuid }}</a>
+                    {% if crash.cpu_info %}
+                        {% if crash.cpu_info.startswith('AuthenticAMD family 20 model 1') or
+                              crash.cpu_info.startswith('AuthenticAMD family 20 model 2') %}
```
Comment (peterbe) [inline webapp-django/crashstats/signature/jinja2/signature/signature_reports.html]: Duplication of code, but considering that signature report is going away, I guess it's fine.

**Comment by peterbe** [comment]
Some nits. Especially the lack of a comment that explains why we're doing this one-off hack.

File: webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html (hunk 1 above)
Comment (jrmuizel) [inline webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html]: family 20 won't have any models added and even if it did they should have the bug fixed.

Comment (peterbe) [inline webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html]: That kinda a knowledge would be great to put in a code comment :)

File: webapp-django/crashstats/signature/jinja2/signature/signature_reports.html (hunk 3 above)
Comment (adngdb) [inline webapp-django/crashstats/signature/jinja2/signature/signature_reports.html]: Signature report is not going away, it's report/list/ that is.

Comment (peterbe) [inline webapp-django/crashstats/signature/jinja2/signature/signature_reports.html]: That's not what I heard. Let's discuss more in London. 
If you think this page is going to stick around, perhaps better make this code into a jinja helper function.

**Comment by adngdb** [comment]
@peterbe comment added

**Comment by peterbe** [comment]
> @peterbe comment added

Did you forget to push?

**Comment by adngdb** [comment]
I pushed to the wrong branch. It's in now.

**Comment by peterbe** [comment]
I still think it's dangerous to have repeated code. That comment and the actual code ought to live in a jinja helper function so it can be shared.

**Comment by adngdb** [comment]
Done!

**Pull request merged**